front/src/applications/operationalStudies/views/v2/ManageTrainScheduleV2.tsx

```diff
@@ -16,16 +16,18 @@ import ItineraryV2 from 'modules/pathfinding/components/Itinerary/ItineraryV2';
 import RollingStock2Img from 'modules/rollingStock/components/RollingStock2Img';
 import { RollingStockSelector } from 'modules/rollingStock/components/RollingStockSelector';
 import { useStoreDataForRollingStockSelector } from 'modules/rollingStock/components/RollingStockSelector/useStoreDataForRollingStockSelector';
+import TimesStops from 'modules/timesStops/TimesStops';
 import { Map } from 'modules/trainschedule/components/ManageTrainSchedule';
 import ElectricalProfiles from 'modules/trainschedule/components/ManageTrainSchedule/ElectricalProfiles';
 import TrainSettings from 'modules/trainschedule/components/ManageTrainSchedule/TrainSettings';
 import { formatKmValue } from 'utils/strings';
 
 const ManageTrainScheduleV2 = () => {
-  const { t } = useTranslation(['operationalStudies/manageTrainSchedule']);
-  const { getOriginV2, getDestinationV2 } = useOsrdConfSelectors();
+  const { t } = useTranslation(['operationalStudies/manageTrainSchedule', 'timesStops']);
```

Comment:
You don't use the timesStops translation in this file
```suggestion
  const { t } = useTranslation(['operationalStudies/manageTrainSchedule']);
```

Reply:
Here I load the 'timesStop' translation json because otherwise the translation doesn't happen when refreshing

Reply:
I tested, and actually you can remove it if you add 't' in the dependencies of your useEffect in TimesStops which formatSuggestOPsToRowOPs (around l70)

Reply:
Done